build-rpmostree.sh

```diff
@@ -11,7 +11,7 @@ set -e
 # Set up RPM build environment
 dnf install -y rpmdevtools rpm-build
 rpmdev-setuptree
-dnf download --source rpm-ostree
+dnf download --source rpm-ostree-2024.3-4.el9_4
```

Comment:
Could we try downloading the latest first, and then default to this specific version?  I am not familiar with how often this version might need to be changed.

Reply:
I'm hoping it doesn't change very often.  As long as we're not seeing a specific failure, there is not really an impetus to upgrade.  Fred committed the change in this patch to upstream and it is 2024.4.  I'm hoping that OL picks that up soon and we can dispense with the whole affair.